IPython/html/static/style/style.min.css

```diff
@@ -8964,6 +9001,13 @@ ul#new-menu {
     header */
   margin-bottom: -1px;
 }
+.header_edit {
+  flex: 0 0 auto;
```

Comment:
This particular `.header_edit` is not really important. The other `header_tree` is important because that allows for a solution of #8181. In this case, since the `/edit` page doesn't even scroll, it shouldn't matter.